torch/_dynamo/utils.py

```diff
@@ -1518,7 +1519,7 @@ def get_fake_value(node, tx, allow_non_graph_fake=False):
                 f"constrain_as_value OR constrain_as_size APIs.  {cause}",
                 case_name="constrain_as_size_example",
             )
-        elif isinstance(cause, torch.utils._sympy.value_ranges.ValueRangeError):
```

Comment:
somehow having `follow_imports=skip` on `mypy-torch.utils._sympy.*` causes mypy to think `ValueRangeError` doesn't exist in the module it's not inspecting. But the error goes away if we explicitly import it